packages/utilities/src/utils/hasType.example.md

```diff
@@ -0,0 +1,34 @@
+The `hasType` method is a utility to easily determine whether a given React component is implemented with a
+specific type. This utility is also able to compare Garden View components that are wrapped in an HOC
+include styled-components, emotion, or fela.
```

Comment:
Oh, I thought they all exposed it the same way. I'll test locally with Fela for sure and update this.

Reply:
Did some digging and it looks like with `react-fela` our `hasType` static pass-through is visible from Fela! So the following:

```jsx
const { createRenderer } = require('fela');
const { Provider, createComponentWithProxy } = require('react-fela');

const FelaLabel = createComponentWithProxy(() => ({
  color: 'red'
}), Label);

<Provider renderer={createRenderer()}>
  <TextField>
    <FelaLabel>Labely label</FelaLabel>
    <Hint>Hint hint</Hint>
    <Input />
    <Message>Example Message</Message>
  </TextField>
</Provider>
```
worked as expected. I tested with emotion and it worked the same as well, but since we don't have any internal consumers using it (that I'm aware of) no need to give people any ideas 😄